### ISSUE TITLE
Collator protocol: connect only to backing group of selected core

### ISSUE THREAD

**Issue by sandreim** [paritytech/polkadot-sdk#5426]
For the MVP we assumed a single collator will author 3 slots per RCB, and needs connect to all backing groups. Ideally we should connect only to the backing group of the core the collation needs be announced to - the committed core index.

**Comment by eskimor** [comment]
Yeah, although as discussed with @skunert it actually makes a lot of sense to keep slot times to at least relay chain slot times and let on collator take care of submitting all PoVs for that slot for all cores. It is bad enough in needing to do a handover every 6s, but even more frequently adds no value whatsover, but just adds overhead.

**Comment by sandreim** [comment]
I wasn't aware we decided to configure aura to give 3 slots for each author, and yes I agree it helps with multiple collators.

**Comment by skunert** [comment]
> I wasn't aware we decided to configure aura to give 3 slots for each author, and yes I agree it helps with multiple collators.

I mean this is just an option we want to support. Its not like we can decide this for anyone (except influence system chains maybe).
